### DISCUSSION TITLE
Render Ninja form after page transition

### DISCUSSION THREAD

**Discussion by rimkio** [swup/swup#942] [Help]
### Describe your problem 🧐

Currently I'm having an error when I go to a page that has a Ninja form, the form cannot be displayed. I have tried many ways but it doesn't work. Can anyone who has encountered this error help me? Thanks a lot
My code:
const swup = new Swup({
            containers: ['#swup', '#wpadminbar'],
            plugins: [
                new SwupA11yPlugin(),
                new SwupScriptsPlugin({
                    head: true,
                    body: true,
                    optin: false

                  }),
                new SwupBodyClassPlugin(),
                new SwupScrollPlugin()
            ]
        });

### What is your question? 🙋‍♀️

How to display Ninja form?

### URL to minimal reproduction 🔗

https://swup.js.org/plugins/js-plugin/

### Checked all these? 📚

- [X] I have read and searched the [official docs](https://swup.js.org)
- [X] I have checked [discussions](https://github.com/orgs/swup/discussions) and [existing issues](https://github.com/issues?q=is:issue+org:swup) for answers to my question
- [X] I agree to follow this project's [Code of Conduct](https://github.com/swup/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have provided all necessary information to the best of my knowledge

**Comment by daun** [comment]
What have you tried so far?

**Reply by rimkio** [reply]
I tried render it by call ajax when transition page. It's only show spinner and not display form

**Reply by daun** [reply]
Sharing your actual code increases your chances of getting help. Either by posting the code here directly or, even better, by posting a link to a [minimal reproduction](https://swup.js.org/other/minimal-reproductions/). People have limited time, and concrete examples make it easier for everybody.
